S3TransferUtilitySample/src/com/amazonaws/demo/s3transferutility/DownloadActivity.java

```diff
@@ -357,12 +366,52 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+
+    private void  requestPermission()
```

Comment:
remove extra whitespace and change style:

```
private void requestPermission() {
```

Reply:
done